yaptide/admin/simulators.py

```diff
@@ -43,7 +43,9 @@ class SimulatorType(IntEnum):
 topas_key = os.getenv('S3_TOPAS_KEY')
 topas_version = os.getenv('S3_TOPAS_VERSION')
 geant_bucket_name = os.getenv('S3_GEANT_BUCKET')
-installation_path = Path(__file__).resolve().parent.parent.parent / 'bin'
+shieldhit_path = Path('/simulators') / 'shieldhit12a' / 'bin'
```

Comment:
I would extract installation path to a click parameter. Hardcoding it to `/simulators` would make it impossible to work with such code locally.